Scripts/Utilities/Editor/ConditionalCompilationUtility.cs

```diff
@@ -30,9 +30,9 @@ namespace ConditionalCompilationUtility
 	/// }
 	/// </summary>
 	[InitializeOnLoad]
-	public class ConditionalCompilationUtility
+	public static class ConditionalCompilationUtility
 	{
-		const string kEnableCCU = "UNITY_CCU";
+		public const string kEnableCCU = "UNITY_CCU";
```

Comment:
Instead of making this public create a static bool property enabled that does the check you do in the test.